patches/v2023.2.0.patch

```diff
@@ -0,0 +1,423 @@
+   <div class="container">
+-    <a class="navbar-brand" routerLink="/" appA11yTitle="{{ 'pageTitle' | i18n: 'Bitwarden' }}">
++    <a class="navbar-brand" routerLink="/" appA11yTitle="{{ 'pageTitle' | i18n: 'Vaultwarden' }}">
+       <i class="bwi bwi-shield" aria-hidden="true"></i>
+     </a>
+     <div class="collapse navbar-collapse">
+@@ -70,7 +70,7 @@
+               <i class="bwi bwi-fw bwi-user" aria-hidden="true"></i>
+               {{ "accountSettings" | i18n }}
+             </a>
+-            <a bitMenuItem href="https://bitwarden.com/help/" target="_blank" rel="noopener">
++            <a bitMenuItem href="https://github.com/dani-garcia/vaultwarden/" target="_blank" rel="noopener">
+               <i class="bwi bwi-fw bwi-question-circle" aria-hidden="true"></i>
+               {{ "getHelp" | i18n }}
+             </a>
+diff --git a/apps/web/src/app/organizations/create/organization-information.component.html b/apps/web/src/app/organizations/create/organization-information.component.html
+index 6029cfd833..04324b7d19 100644
+--- a/apps/web/src/app/organizations/create/organization-information.component.html
++++ b/apps/web/src/app/organizations/create/organization-information.component.html
+@@ -12,7 +12,7 @@
+       <input bitInput type="text" formControlName="name" />
+     </bit-form-field>
+     <bit-form-field class="tw-w-1/2">
+-      <bit-label>{{ "billingEmail" | i18n }}</bit-label>
++      <bit-label>{{ "email" | i18n }}</bit-label>
+       <input bitInput type="email" formControlName="billingEmail" />
+     </bit-form-field>
+     <bit-form-field class="tw-w-1/2" *ngIf="isProvider">
+diff --git a/apps/web/src/app/organizations/settings/account.component.html b/apps/web/src/app/organizations/settings/account.component.html
+index 1d11412bfb..c7911e418e 100644
+--- a/apps/web/src/app/organizations/settings/account.component.html
++++ b/apps/web/src/app/organizations/settings/account.component.html
+@@ -30,7 +30,7 @@
+         />
+       </div>
+       <div class="form-group">
+-        <label for="billingEmail">{{ "billingEmail" | i18n }}</label>
++        <label for="billingEmail">{{ "email" | i18n }}</label>
+         <input
+           id="billingEmail"
+           class="form-control"
+diff --git a/apps/web/src/app/organizations/vault/vault.component.ts b/apps/web/src/app/organizations/vault/vault.component.ts
+index 9519670524..a1a6aa722f 100644
+--- a/apps/web/src/app/organizations/vault/vault.component.ts
++++ b/apps/web/src/app/organizations/vault/vault.component.ts
+@@ -77,11 +77,7 @@ export class VaultComponent implements OnInit, OnDestroy {
+   ) {}
+ 
+   async ngOnInit() {
+-    this.trashCleanupWarning = this.i18nService.t(
+-      this.platformUtilsService.isSelfHost()
+-        ? "trashCleanupWarningSelfHosted"
+-        : "trashCleanupWarning"
+-    );
++    this.trashCleanupWarning = this.i18nService.t("trashCleanupWarningSelfHosted");
```

Comment:
I'm missing the same change for the `trashCleanupWarningSelfHosted` translation for the `apps/web/src/vault/app/vault/vault.component.ts` file.

Reply:
Ah, sorry I think I was tired when checking the differences. I thik I've noticed that the second `trashCleanupWarningSelfHosted` change was missing but I did not notice that the change affected different files. Sorry about that. :facepalm:

I don't think `tw-overflow-x-auto` was missing though, or I am still not seeing it.

Reply:
I'm not seeing the `tw-overflow-x-auto` change in your commit.

Reply:
> I'm not seeing the `tw-overflow-x-auto` change in your commit.

This change is at the bottom of the file (see [line 437](https://github.com/dani-garcia/bw_web_builds/pull/113/files#diff-a9baf2210c79a9297202cf14ded28a154e6174c670211872b6ae595d91abfe6dR437))? Or does it not get applied? I've just tested it to make sure and it seems to work for me so I am a bit confused.

Reply:
Ah, sorry, could have sworn it wasn't there in the first commit though, sorry for that.